tiberius/src/client/connection.rs

```diff
@@ -72,6 +73,26 @@ impl Connection {
         Ok(())
     }
 
+    pub(crate) async fn flush_packets(&mut self) -> crate::Result<()> {
+        poll_fn(|cx| {
+            while let Poll::Ready(Some(packet)) = Pin::new(&mut self.transport).try_poll_next(cx)? {
```

Comment:
What happens if we poll and the stream isn't over (there will be more packets), but the next packet is not there yet?